SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -124,12 +127,12 @@ class USpatialReceiver : public UObject, public SpatialOSDispatcherInterface
 
 	void ApplyComponentDataOnActorCreation(Worker_EntityId EntityId, const Worker_ComponentData& Data, USpatialActorChannel& Channel, const FClassInfo& ActorClassInfo, TArray<ObjectPtrRefPair>& OutObjectsToResolve);
 	void ApplyComponentData(USpatialActorChannel& Channel, UObject& TargetObject, const Worker_ComponentData& Data);
-	
-	// This is called for AddComponentOps not in a critical section, which means they are not a part of the initial entity creation.
-	void HandleIndividualAddComponent(const Worker_AddComponentOp& Op);
+
+	void HandleIndividualAddComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId, TUniquePtr<SpatialGDK::DynamicComponent>&& Data);
```

Comment:
Pass unique_ptr by value unless this function conditionally moves it (which I don't think is the case).

Reply:
Really the dynamic component should contain a unique_ptr but in that case the same advise would apply.